_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
 
 ---
 
+# Important Notes About Bluetooth
+
+Bluetooth is a bit odd. 
+There are a lot of factors that go into whether Bluetooth devices work as expected.
```

Comment:
I would recommend changing this to say something line
`There are a lot of factors that go into whether Bluetooth devices work together as expected.`
This helps the user/reader know that the issue is not just with the Bluetooth hardware of the computer but also the device they are connecting to you.

Reply:
Fixed.